Crear ABM de pedido en controller

zeeno77 opened the pull request:
modified:   src/main/java/com/app/rest/controller/AppController.java	
modified:   src/main/java/com/app/rest/service/ServicioPedido.java

[Aclaracion] Como no estan definidas, presupongo que las funciones de los services siguen el siguiente patron:
get|post|delete + nombre de la funcion del controller.
Si llegan a ser distintas se cambian en el fututo

zeeno77 committed:
modified:   src/main/java/com/app/rest/controller/AppController.java
	modified:   src/main/java/com/app/rest/service/ServicioPedido.java

alangarciac commented on src/main/java/com/app/rest/controller/AppController.java:
```diff
@@ -30,5 +35,22 @@ public String index() {
     }
-    
+    //Devuelve el pedido por id
+    @GetMapping("/pedidos/{id}")
```
Haría renames, antes de seguir tirando codigo, pasando a english. Subí un modelo a seguir (https://trello.com/c/wOy2hkEm/1-soporte-de-bd-mysql) para estar alineados con eso inicialmente, de esta forma `pedido` pasaria a ser > `order`

Este sería el modelo a seguir.

![image](https://user-images.githubusercontent.com/7250456/108419642-8d908000-7211-11eb-81d7-78bb24559920.png)

alangarciac replied:
el get de un element iría en singular:` /order/{id}`

alangarciac commented on src/main/java/com/app/rest/controller/AppController.java:
```diff
@@ -30,5 +35,22 @@ public String index() {
+    }
+
+    //Borra un pedido por id
+    @DeleteMapping("/pedidos/{id}")
```
Lo mismo acá, pasaría a ser `/order/{id}`

alangarciac commented on src/main/java/com/app/rest/controller/AppController.java:
```diff
@@ -30,5 +35,22 @@ public String index() {
+    }
+
+    //Crea un nuevo pedido
+    @PostMapping("/pedidos/nuevo")
```
`/order/new`

alangarciac reviewed (changes requested):
Impecable, antes de seguir tirando codigo, hagamos el refactor a **english**. 

- Creé esta tarea https://trello.com/c/ENmuyS4c/6-refactor-to-english

zeeno77 committed:
Refactor to English:
	modified:   src/main/java/com/app/rest/controller/AppController.java
	new file:   src/main/java/com/app/rest/model/Order.java
	deleted:    src/main/java/com/app/rest/model/Pedido.java
	new file:   src/main/java/com/app/rest/service/OrderService.java
	deleted:    src/main/java/com/app/rest/service/ServicioPedido.java
	new file:   src/main/java/com/app/rest/tests/OrderTest.java
	deleted:    src/main/java/com/app/rest/tests/PedidoTest.java
	modified:   target/classes/com/app/rest/controller/AppController.class
	deleted:    target/classes/com/app/rest/model/Pedido.class
	deleted:    target/classes/com/app/rest/service/ServicioPedido.class
	deleted:    target/classes/com/app/rest/tests/PedidoTest.class

zeeno77 commented:
Terminado el Refactoring a Ingles de todo el Proyecto

alangarciac commented on src/main/java/com/app/rest/model/Order.java:
```diff
@@ -0,0 +1,36 @@
+package com.app.rest.model;
+
+public class Order {
```
Acá creo que Tato lo modeló en su branch, es probable que tenga conflicto.

alangarciac commented:
Loego de resolver conflictos con master, se podría mergear para seguir iterando en un nuevo branch en caso de ser necesario.
